dev/build_prevalidation.py

```diff
@@ -92,6 +92,8 @@ def main():
 
   bom_generator = BomGenerator(options)
   bom_generator.determine_and_tag_versions()
+  bom_generator.determine_and_tag_halyard()
```

Comment:
Seems like adding this would cause problems since we already tag Halyard in https://github.com/lwander/spinnaker/blob/3dc2be0723766d69ce69a59820cc8009ad766d8a/dev/build_prevalidation.py#L91.